charts/fluent-bit/Chart.yaml

```diff
@@ -5,8 +5,8 @@ keywords:
   - logging
   - fluent-bit
   - fluentd
-version: 0.20.10
-appVersion: 1.9.9
+version: 0.20.11
```

Comment:
```suggestion
version: 0.21.0
```
I think this probably warrants a bigger version bump.

Reply:
Sure!